setup.cfg

```diff
@@ -61,3 +53,8 @@ omit =
 
 [coverage:report]
 skip_covered = True
+exclude_also =
```

Comment:
can we put this in the `coverage:run` section? that might be the issue why codecov is not picking up this option - this only affects the output of `coverage html` and the like :)

Reply:
`exclude_also` seems to be only available in `report`, but I actually need to upgrade `coveragepy` as it is a new config value